### DISCUSSION TITLE
Can't calibrate because of shut down

### DISCUSSION THREAD

**Discussion by byungjikroh** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#836] [Q&A]
Hello 

I've used AlDente(v1.19.2) for months to limit battery to 50%. 

Recently battery goes from 50% to 1% suddenly and is recharged.

I figured out that I need to calibrate battery so purchased Pro version to calibrate it. 

Calibration mode needs to go down to battery 15% but Macbook shut down at 55% because of no battery and boot it again. 

After that, of course, AlDente's calibration mode is not done and reset and battery is about 57%.

If I discharge right after this, it shut down again at 55%.

In this situation, how can I calibrate battery properly with using AlDent or not? 

Thanks

**Comment by MatthiasKerbl** [comment]
Hi @byungjikroh,

It sounds like you have a severe case of battery miscalibration.
First, to avoid this in the future, fully charge and discharge your battery at least every two weeks. Furthermore, in such a severe case of miscalibration, AlDentes Calibration Mode does not help currently. It prevents the battery from becoming miscalibrated when used periodically.
To get your battery out of this state, try the following:

1. Charge your battery to 100% and leave it plugged in for 1h+ after it reached 100%.
2. Unplug your MacBook and let it discharge as slowly as possible, the best is to just leave it idle and avoid letting it go to sleep.
3. When the MacBook shuts down again or the battery is empty, repeat steps 1 and 2 as often as necessary to calibrate the battery again and it normally discharges to <5% again. Sometimes, this can take 5+ cycles.

Hope that helps.

Cheers,
Matthias

**Reply by byungjikroh** [reply]
Thanks for your suggestion. 

I'll try what you said and let you know if it doesn't work.
